scripts/gulpfiles/package_tasks.js

```diff
@@ -348,7 +348,6 @@ function packageDTS() {
   return gulp.src(handwrittenSrcs, {base: 'typings'})
       .pipe(gulp.src(`${TYPINGS_BUILD_DIR}/**/*.d.ts`, {ignore: [
 	`${TYPINGS_BUILD_DIR}/blocks/**/*`,
-	`${TYPINGS_BUILD_DIR}/generators/**/*`,
```

Comment:
Shouldn't this still exist if this is where the reexports are coming from?

Reply:
It is being deleted from the `ignore:` clause, thus causing it to be included in what gets copied from `build/` to `dist/`.

Reply:
haha ok sweet sorry about that.